content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,67 @@
+
+The historical alerting framework cannot keep up with monitoring the increasingly large volumes of heterogeneous public health data that organizations like Delphi process. Delphi has increased its volume by 1000x in the past three years alone. And, although Delphi processes data from several geographical regions, even local public health agencies who use systems (that focus on one geography) under the existing alerting framework have challenged its usefulness (e.g. "[What Can You Really Do with 35,000 Statistical Alerts a Week Anyways?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606136/)" (2019)). We anticipate this failure mode for alerting systems will become more prevalent as data volumes grow. 
+
+### Bridging the gap with FlaSH:
+
+Identifying and communicating noteworthy events from public health data is a critical step for future pandemic prevention <sup>[4](#link4)</sup>. But how should we redesign our event detection process? 
+After reflecting on our prior system, we realized the reason why existing monitoring fail, best summarized by the following quote: 
+
+<blockquote>
+"The current disconnect among algorithm developers, implementers, and [experts] … foster[s] distrust in statistical monitoring and in biosurveillance itself."  - Galit Shmeuli & Howard Burkom <sup>[5](#link5)</sup>
+</blockquote>
+    
+Interestingly, through our prior system, I had the opportunity to experience all three roles, from developing the outlier detection methods, implementing and deploying the methods over Delphi's data, and then analyzing the outputs of the methods. These three roles came with different constraints and priorities. These experiences set the foundation for devising an interdisciplinary approach. 
+
+<center>
+![**Fig 2.** FlaSH's Human-in-the-loop-Framework](/blog/2024-01-30-flash-framework/image2.png)</center>
+
+Combining the methods, engineering, and quality assurance perspectives for event detection from large-scale public health streams, we developed a human-in-the-loop approach to address some fundamental challenges in existing alerting systems. Namely, we rank Delphi's recently updated data by how much it warrants reviewer attention. Reviewers can then decide how many data points they want to investigate, which prioritizes a reviewer's time and does not expect them to review a certain number of alerts flagged via a somewhat arbitrary threshold. 
+
+Here's a quick illustration of why the ranking approach works. The most limited resource in this setting is an expert reviewer's time and attention. We want to show this reviewer as many possible events as possible so that they can interpret their data with their known context. 
```

Comment:
```suggestion
Here's a quick illustration of why the ranking approach works. The most limited resource in this setting is an expert reviewer's time and attention. We want to show this reviewer as many potential events as possible so that they can interpret the data with their known context. 
```